### PR TITLE
Add Mohd Shukri Hasan's devcontainer templates

### PR DESCRIPTION
**Pull request by hsm207**
Update with my devcontainer templates repository: https://github.com/hsm207/devcontainer-templates

Adding templates for learning new technologies e.g. qiskit and contributing to OSS projects that have complicated setup steps but no devcontainer support.

### CODE REVIEW

File: _data/collection-index.yml [hunk 1]
```diff
@@ -638,3 +638,8 @@
   contact: https://github.com/lumenpink/devcontainer-features/issues
   repository: https://github.com/lumenpink/devcontainer-features
   ociReference: ghcr.io/lumenpink/devcontainer-features
+- name: Templates by Mohd Shukri Hasan
+  maintainer: Mohd Shukri Hasan
+  contact: https://github.com/hsm207/devcontainer-templates/issues
+  ociReference: ghcr.io/hsm207/devcontainer-templates
```
Comment (eljog) [inline _data/collection-index.yml:645]: Repository link entry is missing.
Please consider adding a README with the details of the features.

This starter template can be helpful https://github.com/devcontainers/feature-starter

**Review by eljog — COMMENTED** [review]
Thank you for your contribution.
I have added a comment

File: _data/collection-index.yml (hunk 1 above)
Comment (hsm207) [reply _data/collection-index.yml:645]: I've added a link to the repository with the relevant READMEs

**Comment by hsm207** [comment]
@eljog have you had a chance to review the changes I made?

**Review by eljog — APPROVED** [review]
⚡

**Pull request merged**